src/views/GovernanceReviewTeam/BusinessCaseReview/index.tsx

```diff
@@ -2,19 +2,27 @@ import React from 'react';
 import { useTranslation } from 'react-i18next';
 import { Link } from 'react-router-dom';
 import { Link as UswdsLink } from '@trussworks/react-uswds';
+// eslint-disable-next-line camelcase
+import { GetGRTFeedback_grtFeedbacks } from 'queries/types/GetGRTFeedback';
```

Comment:
You can rename this import, which will also get around the lint rule. `import { GetGRTFeedback_grtFeedbacks as GrtFeedbacks } from 'queries/types/GetGRTFeedback';`